Makefile

```
@@ -577,6 +577,17 @@ validate-local: image-validators ## Builds validator images and runs validation
 notices: ## Generates THIRD_PARTY_NOTICES.md aggregating every dependency's license
 	@bash tools/generate-notices
 
+.PHONY: notices-check
+notices-check: ## Verifies THIRD_PARTY_NOTICES.md is up to date (run by the merge-gate on dependency changes)
```

checkmake (0.3.2)

[warning] Target body for "notices-check" exceeds allowed length of 5 lines (8).

(maxbodylength)